pkg/bff/auth/auth_test.go

```diff
@@ -63,11 +65,12 @@ func TestClaimsContext(t *testing.T) {
 
 	// Test context with values
 	c.Set(auth.ContextBFFClaims, vclaims.CustomClaims.(*auth.Claims))
-	c.Set(auth.ContextRegisteredClaims, vclaims.RegisteredClaims)
+	c.Set(auth.ContextRegisteredClaims, &vclaims.RegisteredClaims)
 
 	bclaims, err := auth.GetClaims(c)
 	require.NoError(t, err, "could not fetch bff claims")
-	require.Equal(t, "eee784b5-49b3-452e-97d5-1b01e79f5e62", bclaims.VASP)
+	require.Equal(t, "6f0d943d-6cd7-4745-bc9d-6d65e32c70e9", bclaims.OrgID)
+	require.Equal(t, "eee784b5-49b3-452e-97d5-1b01e79f5e62", bclaims.VASP["testnet"])
```

Comment:
Do we actually define the `testnet` and `mainnet` constants anywhere or have we just been relying on being consistent with the spelling and capitalization?

Reply:
https://app.shortcut.com/rotational/story/6417/unify-bff-string-constants-in-config